packages/nocodb/src/lib/db/sql-client/lib/sqlite/SqliteClient.ts

```diff
@@ -1544,22 +1543,18 @@ class SqliteClient extends KnexClient {
         upQuery,
         this.sqlClient
       );
-      //downQuery += alterTablePK(args.originalColumns, args.columns, downQuery);
-
-      if (upQuery) {
-        //upQuery = `ALTER TABLE ${args.columns[0].tn} ${upQuery};`;
-        //downQuery = `ALTER TABLE ${args.columns[0].tn} ${downQuery};`;
-      }
 
-      await Promise.all(
+      if (synchronous) {
```

Comment:
Since `await Promise.all(` is removed the catch part won't handle the exceptions that happened within the map callback function.

On exception it may print `Uncaught Error`.


You can try the following to reproduce the  bug.


```js

(async () => {

  // without promise.await
  try {
    [1,2,3].map(async () =>{
      throw new Error('test')
    })
  }catch(e){
    console.log(e)
  }

  
  // with promise.await
  try {
    await Promise.all([1,2,3].map(async () =>{
      throw new Error('test')
    }))
  }catch(e){
    console.log(e)
  }
```

Reply:
I changed this to a for loop which seems to act same with Promise.all

Reply:
It will run sequentially with for loop

Reply:
looks good now :+1: